src/python/AsyncStageOut/TransferWorker.py

```diff
@@ -193,10 +198,13 @@ def files_for_transfer(self):
         ftscp copyjob per source:destination.
         """
         source_dests = self.source_destinations_by_user()
-        jobs = {}
-        jobs_lfn = {}
-        jobs_pfn = {}
-        jobs_report = {}
+        # Generate dictionary of new jobs.
+        # output is for big files, output_sm is for small files
+        output = {"jobs": {},
+                  "jobs_lfn": {},
+                  "jobs_pfn": {},
+                  "jobs_report": {}}
+        output_sm = output
```

Comment:
> It would increase number of submissions to FTS. Is there any penalty for us/FTS how much FTS jobs we submit?

Yes there are penalties for both since there would be more jobs to manage in ASO and FTS. It is a trade-off. So I would keep it as simple as possible for now to understand the gain of this feature. If the gain is visible then maybe it is the case to spend more efforts to improve the algo.